### ISSUE TITLE
Consider attaching a .sha256.txt file to build results

### ISSUE THREAD

**Issue by paulo-ferraz-oliveira** [erlef/otp_builds#25]
For security purposes as, e.g., Erlang/OTP already does...

**Comment by paulo-ferraz-oliveira** [comment]
@wojtekmach, does this make sense to you? I remember seeing somewhere there was a "goal" about having a similar file, but I don't know if it's still required or was just a thought.

@starbelly, thoughts?

**Comment by wojtekmach** [comment]
checksums are committed to builds/macos-{arch}.txt file using the second workflow, update_builds_txt.yaml.

I am not sure if we need another another file with checksums.

**Comment by paulo-ferraz-oliveira** [comment]
Yeah, maybe there isn't a security verification required, here.

When I discussed this previously with @starbelly we decided to split the SHA (separate file from...) and the CRC (in .txt), but  I can't remember the reasoning. (I believe we also discussed that if a malicious actor could intercept part of the traffic they could potentially intercept all the traffic - what's the builds.txt file different from the .sha256.txt file, really?).

I'll close, for now, but Bryan can still pitch in if he thinks something important's missing.
